Core/StitchCoreSDK/Sources/StitchCoreSDK/StitchError.swift

```diff
@@ -48,6 +48,8 @@ public enum StitchError: Error, CustomStringConvertible {
             return "\(withMessage): \(withCode)"
         case .requestError(let withError, let withCode):
             return "\(withError): \(withCode)"
+        case .requestErrorFull(let error, let code):
```

Comment:
[nit] so is this getting removed before merge?

Reply:
This is going to get removed by the Swift 5 changes